# Distance setting between cameras
# 1: short, 2: medium, 3: long
uint8 requested_baseline